cypress/support/productHelpers.ts

```diff
@@ -0,0 +1,25 @@
+export function addProductByName(productName: RegExp, previousCount: number) {
+  return cy.get('a.prdocutname')
+    .contains(productName)
+    .closest('div.col-md-3')
+    .as('productContainer')
+    .then(() => {
+      cy.get('@productContainer')
+        .find('a.productcart')
+        .click();
+
+      cy.get('header .block_7 span.label.label-orange.font14')
```

Comment:
those are bad selectors, please check: 

https://www.cypress.io/blog/understanding-selectors-in-testing
https://www.browserstack.com/guide/cypress-best-practices